src/include/simeng/Instruction.hh

```diff
@@ -209,7 +209,7 @@ class Instruction {
   BranchType branchType_ = BranchType::Unknown;
 
   /** If the branch target is known at the time of decode, store it. */
```

Comment:
The comment above this sounds better suited to a function call. It may be better as "The branch offset that may be known at the time of instruction decoding. The default value of 0 represents an unknown branch offset."?

Reply:
Sounds good